SofaScoreAcademy/SofaScoreAcademy/ViewController.swift

```diff
@@ -9,98 +9,78 @@ import UIKit
-    
+    private let tableView = UITableView()
+
     override func viewDidLoad() {
         
         super.viewDidLoad()
+        tableView.separatorStyle = .none
         initializeData()
         setupView()
-        addViews()
-        styleViews()
         setupConstraints()
         
     }
-    
+
     private func initializeData() {
         
         exampleEvents = [
             Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", teamOneScore: 1, teamTwoScore: 2, startTime: startTime1, matchStatus: .isFinished),
             Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", teamOneScore: 0, teamTwoScore: 1, startTime: startTime2, matchTime: 36, matchStatus: .inProgress),
             Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", startTime: startTime3, matchStatus: .notStarted),
-            Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", startTime: startTime4, matchStatus: .notStarted),
+            Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", startTime: startTime4, matchStatus: .notStarted)
         ]
         
     }
-    
+
     private func setupView() {
         
         view.backgroundColor = .lightGray
-        view.addSubview(stackView)
+        view.addSubview(tableView)
+
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(MatchViewCell.self, forCellReuseIdentifier: "MatchViewCell")
         
     }
 
-    func addViews() {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         
-        for event in exampleEvents {
-            
-            let matchView = MatchView()
-            matchView.configure(with: event)
-            stackView.addArrangedSubview(matchView)
-            
-        }
+        return exampleEvents.count
         
-        let leagueView = LeagueView()
-        let exampleLeague = League(country: "Spain", leagueName: "LaLiga")
-        leagueView.configure(with: exampleLeague)
-        view.addSubview(leagueView)
     }
-    
-    func styleViews() {
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         
-        stackView.axis = .vertical
-        stackView.distribution = .fillEqually
-        stackView.alignment = .fill
-        stackView.spacing = 10
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "MatchViewCell", for: indexPath) as? MatchViewCell else {
+            return UITableViewCell()
+        }
+        let event = exampleEvents[indexPath.row]
+        cell.configure(with: event)
+        return cell
         
     }
-    
-    func setupConstraints() {
-        
-        stackView.snp.makeConstraints {
-            $0.centerX.centerY.equalToSuperview()
-            $0.width.equalTo(360)
-            
-        }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
```

Comment:
ajde isprobaj jel mozemo imat da si on sam zna visinu i da nam ova metoda uopce ne treba pa mi javi jel to radi